api/app/settings/common.py

```diff
@@ -910,6 +910,9 @@
 )
 TASK_DELETE_RUN_TIME = env.time("TASK_DELETE_RUN_TIME", default="01:00")
 TASK_DELETE_RUN_EVERY = env.timedelta("TASK_DELETE_RUN_EVERY", default=86400)
+RECURRING_TASK_RUN_RETENTION_DAYS = env.int(
+    "RECURRING_TASK_RUN_RETENTION_DAYS", default=30
+)
```

Comment:
Is there a reason not to use the same retention period for both types of task?

Reply:
Yeah, I think we probably want to control this independently, e.g: we can probably go with a lower value here because it's not as critical as tasks?